ios/RCTMarkdownUtils.mm

```diff
@@ -105,10 +106,13 @@ - (NSAttributedString *)parseMarkdown:(nullable NSAttributedString *)input
     } else if ([type isEqualToString:@"blockquote"]) {
       CGFloat indent = _markdownStyle.blockquoteMarginLeft + _markdownStyle.blockquoteBorderWidth + _markdownStyle.blockquotePaddingLeft;
       NSMutableParagraphStyle *paragraphStyle = [NSMutableParagraphStyle new];
-      paragraphStyle.firstLineHeadIndent = indent;
-      paragraphStyle.headIndent = indent;
+      paragraphStyle.firstLineHeadIndent = indent * depth;
+      paragraphStyle.headIndent = indent * depth;
```

Comment:
I've noticed that here we were already using `indent` and you have introduced term `shift` for the same thing. Let's unify them.